data/events.yml

```diff
@@ -0,0 +1,20 @@
+# To add an event in the GitHub user interface, click on the Edit button at the top right which will submit a pull request.
+#
+# Available properties per event
+#  eventId: Should be in format "yyyymmdd-suffix". The yyyymmdd is the date
+#  meetupUrl: (optional) Link the event organization itself.
+#  location: Should be in format "city"
+#  date: Must be in format "yyyy-mm-dd". If multiple days cover our project, only note the first day
+#  talks: (optional) If the CFP is not yet accepted, do not include it yet.
+#    - title: Session title (without the conference or organizer name).
+#      presenters: (optional) Comma separated list of presenters.
+#
+# Order events by date descending
+
+- eventId: 20221124-brussels
+  meetupUrl: https://www.meetup.com/belgian-java-user-group/events/289451067/
+  location: Brussels
+  date: 2022-11-24
+  talks:
+    - title: "How Java 19 influences the future of your high-scale Java Applications"
+      presenters: Ana-Maria Mihalceanu
```

Comment:
Ideally we want to write the presenter's githubId here instead, then add a data/users.yml
like this: https://github.com/kiegroup/optaplanner-website/blob/main/optaplanner-website-root/data/users.yml
and then render it with this macro: https://github.com/kiegroup/optaplanner-website/blob/main/optaplanner-website-root/templates/macros.ftl#L169
to get something like this 
![image](https://user-images.githubusercontent.com/176880/203929496-1d11c74f-aa2f-4792-8c4c-c2be43d0a093.png)